server/index.js

```diff
@@ -1,7 +1,9 @@
-var express = require('express')
-var app = express()
-app.get('/', function (req, res) {
-  res.send('hello world')
-})
-
-app.listen(3000)
+import express from 'express';
+import connectDb from './mongo.js';
+const app = express();
+ 
+connectDb().then(async () => {
+  app.listen(3000, () =>
```

Comment:
please store this port value into config file and then use it from there

Reply:
Done